src/config.rs

```diff
@@ -21,7 +21,7 @@ pub struct Config {
 #[derive(Deserialize, Clone)]
 pub struct Group {
     pub group_name: String,
-    pub cycle_time_ms: u16,
+    pub cycle_time_μs: u16,
```

Comment:
If I understand this correctly u16 similar to uint16 allows a max value of 65_535 (https://doc.rust-lang.org/std/u16/constant.MAX.html) this feels like it's not enough for an interval specified in microseconds?

Reply:
FIxed